### ISSUE TITLE
A1 7.1.3 – number of rows in weather dataset

### ISSUE THREAD

**Issue by mathiasbruun** [abjer/sds2019#24]
Hello,

In exercise 7.1.3 of Assignment 1 we are asked to:
> merge station locations onto the weather data spanning 1864-1867.

In the previous exercise, these datasets are combined to form a DataFrame with 30003 rows. However, the assert statement in 7.1.3 goes like this:
```python
assert answer_73.shape == (5686, 15)
```
which seems to indicate that we are supposed to merge location data with only one dataset (the one from 1864?).

Maybe I'm just going crazy, but i hopefully someone can clarify! :)

/ Mathias

**Comment by Kristianuruplarsen** [comment]
Good catch. The way the question reads it should obviously be asserting that there are 30003 rows. I have pushed a new version of the assignment which allows both answers, i.e. using data for only 1864 or for 1864-67. 

It turned out in A0 that the autograder can handle older versions of the assignments just fine, so you only need to get the new version if you want to solve the problem using data for 1864-67.  Otherwise just solve the exercise using only the 1864 data (in which case the test should pass).

**Comment by mathiasbruun** [comment]
Got it, thanks!
